specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerDefaultRuleCollectionPost.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager",
+    "configurationName": "myTestSecurityConfig",
+    "parameters": {
+      "networkGroupIdList": [
+        {
+          "networkGroupId": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/networkGroups/testGroup"
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/securityConfigurations/myTestSecurityConfig/ruleCollections/DefaultRuleCollection",
+        "name": "testRuleCollection",
+        "type": "Microsoft.Network/networkManagers/securityConfigurations/RuleCollections",
+        "systemData": {
+          "createdBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+          "createdByType": "User",
+          "createdAt": "2021-01-11T18:52:27Z",
+          "lastModifiedBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2021-01-11T18:52:27Z"
+        },
+        "properties": {
+          "displayName": "testDefaultRuleCollection",
+          "description": "A sample default rule collection",
+          "provisioningState": "Succeeded",
+          "securityType": "UserPolicy",
```

Comment:
securityType will be a redundant field if we associate the RuleCollection under particular configuration which is already scoped to either "AdminPolicy"|"UserPolicy"